stdlib/record.ncl

```diff
@@ -107,5 +107,93 @@
         else
           r in
         %record_insert% field r content,
+    
+    map_values: forall a b. (a -> b) -> {_: a} -> {_: b}
+    | doc m%"
+      Maps a function over every field value of a record.
+
+      For example:
+      ```nickel
+        map_values (fun x => x + 1) { hi = 2 } =>
+          { hi = 3 }
+        map (fun x => x + 1) { hello = 1, world = 2 } =>
+          { hello = 2, world = 3 }
+      ```
+    "%
+    = fun f => map (fun _field value => f value),
+
+    as_array : forall a. {_: a} -> Array {field: Str, value: a}
```

Comment:
`to` is more in line with the naming elsewhere in Nickel? Maybe?

```suggestion
    to_array : forall a. {_: a} -> Array {field: Str, value: a}
```

Reply:
`to_array` also feels more complementary to `from_array` 👍